fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1168,6 +1170,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static boolean tablet_sched_disable_colocate_overall_balance = false;
 
+    @ConfField(mutable = true)
+    public static long[] tablet_sched_colocate_balance_high_prio_backends = {};
+
+    @ConfField(mutable = true)
```

Comment:
What's the usage of this configuration?

Reply:
for airbnb case, to make decommissioned replica not waitting for txn watermark, i think it can be useful in other uses' senario